templates/emails/failed-preorder-authentication.php

```diff
@@ -15,8 +15,8 @@
 <?php
 	echo wp_kses(
 		sprintf(
-			// translators: %s is a link to the payment re-authentication URL.
-			_x( 'Your pre-order is now available, but payment cannot be completed automatically. %s', 'In failed SCA authentication for a pre-order.', 'woocommerce-gateway-stripe' ),
+			// translators: 1) is a link to the payment re-authentication URL.
```

Comment:
Just wanted to make a note that we don't need to update this file at this point. It's updated out-of-the-box in the build/release process.